integration/spark-common/src/main/scala/org/apache/spark/sql/catalyst/CarbonDDLSqlParser.scala

```diff
@@ -932,6 +958,11 @@ abstract class CarbonDDLSqlParser extends AbstractCarbonSparkSQLParser {
     }
   }
 
+  def isEscapeSequence(loadOption: String): Boolean = {
```

Comment:
Rename it to isValidEscapeSequence and also add one valid test  case for validating a correct escape sequence